solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -225,16 +224,19 @@ function add_image_info() {
 }
 
 function add_node_info_solution_config() {
-echo "Updating node info in solution.yaml"    
+    add_secondary_separator "Updating node info in solution.yaml"    
 
     pushd $SCRIPT_LOCATION/k8_cortx_cloud
-        count=1
+        if ! grep -q nodes solution.yaml; then
+        count=0
         for node in $(kubectl get nodes -o jsonpath="{range .items[*]}{.metadata.name} {.spec.taints[?(@.effect=='NoSchedule')].effect}{\"\n\"}{end}" | grep -v NoSchedule)
             do
-            i=$node yq e -i '.solution.nodes['$count'].node'$count'.name = env(i)' solution.yaml
+            i=$node yq e -i '.solution.storage_sets[0].nodes['$count'] = env(i) ' solution.yaml
             count=$((count+1))
-        done
-        sed -i -e 's/- //g' -e '/null/d' solution.yaml
+            done
+        else
+        add_secondary_separator "Nodes entries already present in solution.yaml. Skipping update"
```

Comment:
please indent line 237-239 correctly

Reply:
Fixed and tested - https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/RGW/job/setup-cortx-rgw-cluster/9172/console